src/api.js

```diff
@@ -1,26 +1,25 @@
-const API_URL = 'https://pokeapi.co/api/v2/';
+const BASE_URL = 'https://pokeapi.co/api/v2/';
 
-// make an API call to 'PokeAPI'
-const apiCall = async ( endpoint ) => {
-	return fetch( API_URL + endpoint )
-		.then( ( res ) => res.json() );
+// Make a GET request to 'PokeAPI'.
+const get = async ( endpoint ) => {
+	return fetch( BASE_URL + endpoint ).then( ( res ) => res.json() );
 };
 
-// fetch all pokemons
+// Fetch all pokemons.
 export const fetchPokemons = async ( limit = 151, offset = 0 ) => {
-	return apiCall( `pokemon?limit=${ limit }&offset=${ offset }` );
+	return get( `pokemon?limit=${ limit }&offset=${ offset }` );
 };
 
-// fetch specific pokemon data
+// Fetch specific pokemon data.
 export const fetchPokemonData = async ( pokemonId ) => {
-	return apiCall( `pokemon/${ pokemonId }` );
+	return get( `pokemon/${ pokemonId }` );
 };
 
-// fetch pokemon evolutions
+// Fetch pokemon evolutions.
 export const fetchPokemonEvolutionChain = async ( pokemonId ) => {
-	return apiCall( `pokemon-species/${ pokemonId }` ).then( ( data ) => {
+	return get( `pokemon-species/${ pokemonId }` ).then( ( data ) => {
 		const evolutionChainId = data.evolution_chain.url.match( /\/(\d+)\// )[ 1 ];
 
-		return apiCall( `evolution-chain/${ evolutionChainId }` );
+		return get( `evolution-chain/${ evolutionChainId }` );
```

Comment:
Consider adding a comment that explains the regex structure.
Are you sure that there will always be 2 values in the match result? (or do you want to fail in case there aren't?)